config/checkstyle/checkstyle.xml

```diff
@@ -230,12 +230,13 @@
         </module>
         <module name="JavadocMethod">
             <property name="scope" value="public"/>
```

Comment:
![dg3](https://user-images.githubusercontent.com/60384540/97528779-7838e100-19e9-11eb-9f8d-4084dccffaa6.PNG)

The circle highlighted portion for the sequence design is very small and hard to see the different components/operations. It would improve the visusal better if more spaces are given to the main ravi box for the sequence diagram.